index.html

```diff
@@ -953,14 +953,141 @@ <h3>
+
+          This is due to the information disclosure that happens as part of the request,
+          and the ability for wallets to retain or share this information at time of the
+          request.
+        </p>
+
+        <h4>
+          Permission vs. Consent
+        </h4>
+        <p>
+          The permission mediated by the user agent is not consent, which can vary among
+          different legal and regulatory environments and is expected to be collected by
+          the digital wallet before sharing information with the verifier.
+
+          The exchange protocol is responsible for including relevant information for consent,
+          possibly with additional disclosures such as:
+        </p>
+
+        <ul>
+        <li>
+          The privacy policy of the verifier receiving the credential.
+        </li>
+        <li>
+          The purpose of the verifier for requesting the information.
+        </li>
+        <li>
+          Assertions of the verifier's legimitacy and registration for accessing
+          the credential, such as <a href="https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/annexes/annex-2/annex-2-high-level-requirements.md#a2327-topic-27---registration-of-pid-providers-providers-of-qeaas-pub-eaas-and-non-qualified-eaas-and-relying-parties">EUDI access and registration certificates</a>.
+        </li>
+        </ul>
+
+        <p>
+          However, in practice, it is difficult for user agents to assert that this disclosure
+          is universally being made to users, and to judge the quality of the presentation to the user.
+          User agents could improve independently ensure that a sensible level of transparency is
+          upheld through a number of interventions:
+        </p>
+
+        <ul>
+        <li>
+          Requiring relevant fields such as a Privacy Policy to be passed in the
+          credential request metadata of the exchange protocol.
+        </li>
+        <li>
+          Additionally, including the relevant provided information in the user agent permission prompt.
+        </li>
+        <li>
+          Auditing and approving verifier websites only if they meet a certain standard
+          of user disclosure.
+        </li>
+        <li>
+          Auditing and approving wallet applications only if they meet a certain standard
+          of user disclosure.
+        </li>
+        </ul>
+
+        <p>
+          All of these options have drawbacks, e.g. increasing effort and process to
+          participate in credential exchanges on the Web, and are dependent on the evolution of
+          other initiatives and legal or regulatory frameworks in this problem space.
+          We expect future efforts in the relevant working groups to
+          ensure that the Digitial Credentials API empowers all user agents to collaborate
+          effectively to improve user transparency, permission and consent.
```

Comment:
I don't know what this means. 

Also, permission and consent typically aren't the kinds of concepts that you improve; do you mean something more specific -- increasing the likelihood that users are able to give informed consent, making permission requests more comprehensible or less burdensome, etc.?

Reply:
I can try to reframe this!